docker-compose.yml

```diff
@@ -14,6 +14,16 @@ services:
       quorum_net:
         ipv4_address: '172.14.0.99'
 
+  eth-stats:
+    build: https://github.com/lucashenning/eth-netstats.git
```

Comment:
Awesome stuff, didn't know you could include a git repo directly!

Could you update this to `https://github.com/DeloitteBlockchain/eth-netstats.git` (we forked your great repo, just to have it all under `DeloitteBlockchain` account).